tools/dev.sh

```
@@ -0,0 +1,45 @@
+# Dev server script — runs runserver + dispatcherd + scheduler with auto-reload.
+# Unlike `metrics_service run` (which uses gunicorn), runserver picks up
+# template and code changes without a restart.
+
+set -euo pipefail
+
+cd "$(dirname "$0")/.."
+
+MANAGE="uv run python manage.py"
+
+# --- init ---
+echo SKIPPED: $MANAGE migrate
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)